docs/preview/02-Features/04-AzureActiveDirectory/client-secret-expiration-job.md

```diff
@@ -23,30 +23,26 @@ If this is the case either a `ClientSecretAboutToExpire` or `ClientSecretExpired
 
 ## Usage
 
-Our background job has to be configured in `ConfigureServices` method:
+The background job can easily be added to any .NET hosted application. It is recommanded, though, to create a dedicated background application to run background jobs.
```

Comment:
```suggestion
The background job can easily be added to any .NET hosted application. It is recommended though to create a dedicated background application to run background jobs.
```
small typo :)
And I think the comma's are not required.

Reply:
Hihi, I almost always write 'recommended' wrong 😄 .